cvat-ui/src/components/model-new-annotation-modal/styles.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+@import '../../base.scss';
+
+.cvat-run-model-dialog > div:not(first-child) {
+    margin-top: 10px;
+}
+
+.cvat-run-model-dialog-remove-mapping-icon {
+    color: $danger-icon-color;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5609236)